src/javaharness/.bazelrc

```diff
@@ -13,5 +13,5 @@ build --watchfs
 build --spawn_strategy=local
 build --strategy=J2cl=worker
 build --strategy=Closure=worker
-build --experimental_persistent_javac
+#build --experimental_persistent_javac
```

Comment:
Why comment out? Maybe add a comment explaining?

Reply:
reverting, i was trying to debug a compiler crash